landoui/templates/stack/stack.html

```diff
@@ -142,12 +142,14 @@ <h2>Landing is blocked:</h2>
           </button>
           <button class="StackPage-landingPreview-close button">Cancel</button>
         </form>
-        <form class="StackPage-landingPreview-uplift" action="{{ url_for('revisions.uplift') }}" method="post">
-          {{ uplift_request_form.csrf_token }}
-          {{ uplift_request_form.landing_path }}
-          {{ uplift_request_form.repository }}
-          <button class="button">Request uplift</button>
-        </form>
+        {% if is_api_token_available() %}
+            <form class="StackPage-landingPreview-uplift" action="{{ url_for('revisions.uplift') }}" method="post">
```

Comment:
This is more for an overarching theme/comment and out of scope for this PR, but for new functionality I think it would be better to start using async requests to the API (and handling the response with JavaScript -- event if it's redirecting/refreshing the page.) E.g. similar to how the cancel button works. Lando will be going in that direction eventually as a whole.